src/sidebar/helpers/annotation-metadata.ts

```diff
@@ -394,6 +394,14 @@ export function quote(annotation: APIAnnotationData): string | null {
   return quoteSel ? quoteSel.exact : null;
 }
 
+/** Return the description of the annotation's selection. */
+export function description(annotation: APIAnnotationData): string | undefined {
+  if (annotation.target.length === 0) {
+    return undefined;
+  }
+  return annotation.target[0].description;
```

Comment:
I reckon this could be simplified:

```suggestion
  return annotation.target[0]?.description;
```